web/docs/contribute/notebooks.md

```diff
@@ -34,6 +34,21 @@ Since the `web` directory is a Quarto
 [project](https://quarto.org/docs/projects/quarto-projects.html), it suffices
 there to run `quarto render` to generate all contained notebooks.
 
+## Run within Docker
+
+We also provide a Docker environment to enable a reproducible execution of
+Quarto commands (including its dependencies such as R). This images is built on
+top of the VAST image, and thus contains a working binary that can be used to
+demonstrate VAST features.
+
```

Comment:
- I am confused with your [previous comment](https://github.com/tenzir/vast/pull/2625/files/7e660d81f937dc6c2cfd2c6319014f48289e130e#r1001876980), what should the final text be??? 
- actually this is not accurate: currently in the web Makefile the `vast` service is not started, only Quarto. I can change that if you want, but my original approach was to add extra services only if they are actually used in the docs.

Reply:
Yep, both was the intention!

Reply:
We should probably mention that docs writers can either just have a single VAST container, or use Docker Compose for E2E examples. No?

Reply:
let's trackback just a little bit on what this PR actually enables or not:
- we have the compose overlays, which by themselves are pretty hard to use even if we provide some examples in the associated README
- we provide a `Makefile` target `docker` in `examples/notebooks` that spins up both the `vast` and the `quarto` service, and a sample notebook that just runs  `vast status` to show (how) it works (which outputs to `html` instead of `md`  for a reason I can't explain)
- we provide a `Makefile` target `docker` in `web` that only spins up `quarto` to build the documentation notebooks

This is not a definitive feature set, I would expect the environment in which the documentation and examples are built to get ever richer, with first the addition of the `vast` service in the documentation environment when required, then probably other services. But I would argue that this enrichment of the environment should be added incrementally according to the needs of subsequent PRs. I don't really know how we should reflect that in the documentation.

Reply:
I'm in agreement with this approach. It sounds like we're actually saying the same. There's strict increase in complexity, starting with just a Docker image, and then there will be various add-ons, each in the form of another Docker Compose service.

The scope of this PR is stopping with the `quarto` and `vast` services.

Reply:
(Feel free to resolve this thread.)

Reply:
I added a small paragraph regarding extensibility of the docker compose context because i think it's important:
```
Other services can be added to the context of the Quarto notebook execution by
extending the Docker Compose setup with [extra
overlays](https://github.com/tenzir/vast/tree/master/docker/).
```